### ISSUE TITLE
Web framework attaches elements to the DOM with disregard for screen readers.

### ISSUE THREAD

**Issue by coreygreenwald** [airbnb/lottie-web#2503]
**Tell us about your environment**
All environments using Talkback for Accessibility

* **Browser and Browser Version:**
Environment Agnostic. Works with Talkback (Android) and VoiceOver (iOS)

* **After Effects Version:**
n/a

**What did you do? Please explain the steps you took before you encountered the problem.**
1. Enable Talkback/Voiceover on your platform of choice.
2. Navigate to a web page consuming a lottie animation.
3. Advance Talkback to the end of content and wait to hear "giItT1WQy@!-/#"

I had Talkback/VoiceOver on and I am able to hear elements mounted to the DOM for use by Lottie that are not for Human consumption. (creation of nodes - see attached)

![Screen Shot 2021-03-30 at 11 05 55 AM](https://user-images.githubusercontent.com/17935662/113011701-0a374800-9148-11eb-8a82-13b21315549e.png)

**What did you expect to happen?**
Talkback/Voiceover to end when the physical content on the DOM ended.

**What actually happened? Please include as much _relevant_ detail as possible.**
Talkback/Voice over confuses those who are visually impaired by announcing elements that are not visible to the eye the number of times the node exists ~10+ times. The parent element should be set with the attribute `aria-label="hidden"` to stop this problem but not affect the node's functionality. 

**Please provide a download link to the After Effects file that demonstrates the problem.**
n/a
